web-animations-2/Overview.bs

```diff
@@ -2460,13 +2480,42 @@ given values of <var ignore=''>behavior</var>:
 
 Issue: Do we need a proper definition for "behavior types primary/inverse"?
 
-### Animation Trigger Active Interval ### {#trigger-active-interval}
+#### Event-Based Animation Trigger Behavior #### {#event-trigger-behaviors}
+
+For [=event-based animation triggers=] changes to animation playback happen when a
+qualifying event occurs, according to the trigger's [=animation trigger behavior=]
+and the current [=play state=] of the associated [=animation=]:
```

Comment:
I suspect that the result of #12119 may be that we can share more of this functionality, though I suppose it would be fine to merge the two behavior sections afterwards. Let's try to align though on similar behaviors as much as it makes sense. @DavMila

Reply:
I think this partly relates to https://github.com/w3c/csswg-drafts/issues/11914.

If we specify that playback methods disable triggers, then the trigger doesn't need to take Animation.playState into account and we can write our expectations for a `once` trigger (event- or timeline- based) just in terms of the trigger's state, i.e. a `once` trigger plays its animation when it transitions from `idle` to `primary` (which can only happen once). 

If they don't disable  triggers, then we might want triggers to be a little smarter and perhaps it makes sense to factor in the animation's playState. I think the logic would still be the same for event-based and timeline-based.

Reply:
My intention here is that event-based triggers do not have a "trigger state". I don't even think it makes sense to have an "idle" pre-period for event-based triggers, it's just a bad conceptual fit.

I'm also not sure it makes sense to disable an event-based trigger whenever a WAAPI method is called on an animation. I need to mull that over a bit, but my first instinct is that it would be confusing/surprising.